examples/axisartist/demo_curvelinear_grid.py

```diff
@@ -27,13 +27,8 @@ def curvelinear_test1(fig):
     Grid for custom transform.
     """
 
-    def tr(x, y):
-        x, y = np.asarray(x), np.asarray(y)
-        return x, y - x
-
-    def inv_tr(x, y):
-        x, y = np.asarray(x), np.asarray(y)
-        return x, y + x
+    def tr(x, y): return x, y - x
+    def inv_tr(x, y): return x, y + x
```

Comment:
Style: I don't think we should put function bodies on the same line (and I'm surprised flake8 doesn't complain). Either give it proper line breaks (preferred), or inline them into `GridHelperCurveLinear using lambdas.

Reply:
One liners can be written together, see https://www.python.org/dev/peps/pep-0008/#blank-lines / https://github.com/PyCQA/pycodestyle/pull/823.  I can still spread things out if you prefer, just let me know.